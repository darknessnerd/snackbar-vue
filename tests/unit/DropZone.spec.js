import { shallowMount } from '@vue/test-utils';
import DropZone from '@/DropZone.vue';

describe('DropZone.vue', () => {
  beforeEach(() => {
    jest.resetModules();
    jest.clearAllMocks();
  });
  it('renders correctly', () => {
    const wrapper = shallowMount(DropZone,
      {
        attachTo: document.body,
      });
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
