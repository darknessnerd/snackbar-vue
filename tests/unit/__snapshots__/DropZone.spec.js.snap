// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropZone.vue renders correctly 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "__emits": Object {},
      "__props": Array [
        Object {},
        Array [],
      ],
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <form
        class="dropzone dropzone__box dropzone-clickable"
      >
        <div
          class="dropzone__message dropzone__message--style dropzone-clickable"
        >
          
          Drop here
          
        </div>
        
        
      </form>
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {
        "transition": Object {
          "name": "transition",
          "props": undefined,
          "render": [Function],
        },
        "transition-group": Object {
          "name": "transition-group",
          "props": undefined,
          "render": [Function],
        },
      },
      "config": Object {
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isCustomElement": [Function],
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "mixins": Array [
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
      ],
      "provides": Object {},
      "reload": [Function],
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isCustomElement": [Function],
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.0.6",
  },
  "__setProps": [Function],
  "componentVM": Object {
    "acceptedFiles": null,
    "accepts": Array [],
    "all": Object {},
    "chunking": false,
    "clickable": true,
    "dropzone": <form
      class="dropzone dropzone__box dropzone-clickable"
    >
      <div
        class="dropzone__message dropzone__message--style dropzone-clickable"
      >
        
        Drop here
        
      </div>
      
      
    </form>,
    "dropzoneClassName": "dropzone__box",
    "dropzoneDetailsClassName": "dropzone__details--style",
    "dropzoneItemClassName": "dropzone__item--style",
    "dropzoneMessageClassName": "dropzone__message--style",
    "filesize": [Function],
    "handleDragOver": [Function],
    "headers": null,
    "hiddenInputContainer": "body",
    "ids": Array [],
    "maxFileSize": 1000000,
    "maxFiles": null,
    "method": "POST",
    "multipleUpload": false,
    "numberOfChunks": 10,
    "onDrop": [Function],
    "parallelUpload": 3,
    "paramName": "file",
    "processQueue": [Function],
    "removeFile": [Function],
    "retryOnError": false,
    "uploadOnDrop": true,
    "url": null,
    "withCredentials": false,
    "xhrTimeout": 60000,
  },
  "rootVM": Object {},
}
`;
